internal/sharedgate/sharedgate.go

```diff
@@ -18,7 +18,7 @@ package sharedgate // import "go.opentelemetry.io/collector/internal/sharedgate"
 import "go.opentelemetry.io/collector/featuregate"
 
 var ConnectorsFeatureGate = featuregate.GlobalRegistry().MustRegister(
-	"enableConnectors",
+	"connectors",
```

Comment:
I think `service.connectors` is more appropriate?

Reply:
Updated.